src/main/scala/fitnesse/idea/lexer/FitnesseLexer.scala

```diff
@@ -229,4 +231,8 @@ class LexerSourcePage extends SourcePage {
   override def compareTo(o: SourcePage): Int = {
     throw new IllegalStateException("FitNesse plugin: method LexerParsingPage.compareTo() has not been implemented")
   }
+
+  override def getSymbols(symbolType: SymbolType): util.List[Symbol] = {
```

Comment:
Why has this method been added?

Reply:
Because it was added (in a recent fitnesse release) to `SourcePage` interface that this class implements.
So otherwise it does not compile, causing the build failure of my first push.

Reply:
👍